doc/cephfs/add-remove-mds.rst

```diff
@@ -62,7 +62,13 @@ means limiting its cache size.
 Adding an MDS
 =============
 
-#. Create an mds data point ``/var/lib/ceph/mds/ceph-${id}``. The daemon only uses this directory to store its keyring.
+.. note::
+   It is highly recommended to use `cephadm`_ for setting up the ceph cluster
```

Comment:
```suggestion
   It is highly recommended to use `cephadm`_ or another Ceph orchestrator for setting up the ceph cluster
```

Reply:
I've added the line, thanks for your input!